### ISSUE TITLE
Accessibility issue: Text on image contrast is too low

### ISSUE THREAD

**Issue by mennovogel** [parley-messaging/android-library#91]
From our accessibility audit we got the following issue:

_The contrast ratio between the color of the normal text and the color of its background is too low. Text smaller than 18pt (24px) in regular font weight or 14pt (18.66px) in bold font is considered normal text. Ensure that all normal text has a contrast ratio of at least 4.5:1 against its background.

The date "15:20" does not have a fixed background color. Because the image is yellow it only has a contrast of 2:1._

![Image](https://github.com/user-attachments/assets/ab373495-e687-4a25-b62b-db7f856c5004)

I know that there is a shadow behind the text on the image, but in some cases this shadow is not enough to create enough contrast.

The version of the library was 3.11.2.

**Comment by mennovogel** [comment]
Thanks, looks good.

**Comment by alexkok** [comment]
Fixed in [3.11.3](https://github.com/parley-messaging/android-library/releases/tag/3.11.3) 🚀 

(make sure to set the style for this to apply it)

**Comment by mennovogel** [comment]
Hi @alexkok ,

Thank you for the quick fix. Sorry for my quick approval.

Unfortunately, the checkmark also is required to have the background as it communicates something to the user. Would you please be able to extend the background to the checkmark?

In addition, the designers would like to be able to give the corners a curve radius.  (this is not an accessibility requirement)

**Comment by alexkok** [comment]
Thanks @mennovogel , will reopen to add this 👍

**Comment by alexkok** [comment]
Released in [3.11.4](https://github.com/parley-messaging/android-library/releases/tag/3.11.4) 🚀
